### PR TITLE
DT-2976: 🔨 Upgrade to latest springfox

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/resource/api/UserController.kt [hunk 1]
```diff
@@ -217,23 +217,23 @@ class UserController(private val userService: UserService) {
...
+      dataType = "java.lang.Integer",
       paramType = "query",
       value = "Number of records per page.",
       example = "10",
       defaultValue = "10"
     ),
     ApiImplicitParam(
       name = "sort",
-      dataType = "string",
+      dataType = "java.lang.Integer",
```
Comment (SimonMitchellMOJ) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/resource/api/UserController.kt:236]: Should this still be a string?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/resource/api/AuthUserController.kt [hunk 2]
```diff
@@ -147,23 +147,23 @@ class AuthUserController(
...
+      dataType = "java.lang.Integer",
       paramType = "query",
       value = "Number of records per page.",
       example = "10",
       defaultValue = "10"
     ),
     ApiImplicitParam(
       name = "sort",
-      dataType = "string",
+      dataType = "java.lang.Integer",
```
Comment (SimonMitchellMOJ) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/resource/api/AuthUserController.kt:166]: Should this be` java.lang.string`?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/resource/api/UserController.kt (hunk 1 above)
Comment (petergphillips) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/oauth2server/resource/api/UserController.kt:236]: :facepalm:

**Review by SimonMitchellMOJ — APPROVED** [review]
LGTM

**Pull request merged**